test/system/current_user_test.rb

```diff
@@ -217,4 +217,46 @@ class CurrentUserTest < ApplicationSystemTestCase
 
     assert_text 'textbringer'
   end
+
+  test 'update admin user\'s auto_retire' do
+    user = users(:komagata)
+
+    # auto_retireはデフォルトでtrue
+    # チェックボックスにチェックを入れると、auto_retireはfalseになる
+    visit_with_auth '/current_user/edit', 'komagata'
+    check '休会六ヶ月後に自動退会しない', allow_label_click: true
+    click_on '更新する'
+
+    assert_not user.reload.auto_retire
+  end
+
+  test 'update admin user\'s invoice_payment' do
+    user = users(:komagata)
+
+    visit_with_auth '/current_user/edit', 'komagata'
+    check '請求書払いのユーザーである', allow_label_click: true
+    click_on '更新する'
+
+    assert user.reload.invoice_payment
```

Comment:
```suggestion
    visit_with_auth '/current_user/edit', 'komagata'
    check '請求書払いのユーザーである', allow_label_click: true
    click_on '更新する'

    assert users(:komagata).reload.invoice_payment
```

一度しか出てこないのであれば変数に代入する必要はないかもです。